vignettes/vroom.Rmd

```diff
@@ -112,6 +112,12 @@ file <- vroom_example("mtcars.csv.gz")
 vroom(file)
 ```
 
+About how it work `vroom()`. After the works of this function (indexing and writes) the already uncompressed file is going to compressed again but into a temporal file (in a `tempdir()`).
```

Comment:
I think you can collapse the first fragment with the second full-sentence (see suggestion)

Reply:
What you mean with that?

Reply:
"About how it work `vroom()`" isn't actually a full sentence. I think it's implied from the surrounding context and language that you're talking about how the function works, so in my suggestion, I brought the mention of `vroom()` into that first full sentence (second, if you're counting the initial frag!)

Reply:
So, I think this is resolved, since you accepted those edits!